[CI] TranslogTests.testStats() Failure

**Build scan**: https://gradle-enterprise.elastic.co/s/yqjupuaexvuaq

**Repro line**: `./gradlew ':server:test' --tests "org.elasticsearch.index.translog.TranslogTests.testStats" -Dtests.seed=EE716E3D0A4D7ACE -Dtests.security.manager=true -Dtests.locale=bg -Dtests.timezone=Eire -Druntime.java=11`

**Reproduces locally?**: No (but haven't tried to reproduce it on windows)

**Applicable branches**: master

**Failure history**: Failed once in last 7 days

https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=Europe/Amsterdam&tests.container=org.elasticsearch.index.translog.TranslogTests&tests.sortField=FAILED&tests.test=testStats&tests.unstableOnly=true

**Failure excerpt**:

```

java.lang.AssertionError: |  
-- | --
  | Expected: a value greater than <0L> |  
  | but: <0L> was equal to <0L> |  

at __randomizedtesting.SeedInfo.seed([EE716E3D0A4D7ACE:74F037826291B4ED]:0) |  
-- | --
  |   | at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18) |  
  |   | at org.junit.Assert.assertThat(Assert.java:956) |  
  |   | at org.junit.Assert.assertThat(Assert.java:923) |  
  |   | at org.elasticsearch.index.translog.TranslogTests.testStats(TranslogTests.java:491)

```

--- a/server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java
+++ b/server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java
@@ -483,12 +483,13 @@ public class TranslogTests extends ESTestCase {
         translog.getDeletionPolicy().setLocalCheckpointOfSafeCommit(randomLongBetween(3, Long.MAX_VALUE));
         translog.trimUnreferencedReaders();
         {
+            long lastModifiedAge = System.currentTimeMillis() - translog.getCurrent().getLastModifiedTime();
             final TranslogStats stats = stats();
             assertThat(stats.estimatedNumberOfOperations(), equalTo(0));
             assertThat(stats.getTranslogSizeInBytes(), equalTo(firstOperationPosition));
             assertThat(stats.getUncommittedOperations(), equalTo(0));
             assertThat(stats.getUncommittedSizeInBytes(), equalTo(firstOperationPosition));
-            assertThat(stats.getEarliestLastModifiedAge(), greaterThan(0L));
+            assertThat(stats.getEarliestLastModifiedAge(), greaterThanOrEqualTo(lastModifiedAge));
         }
     }
 

